### ISSUE TITLE
Landmarks issues

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2236]
Referring to #2033.

Two issues with the landmarks structure. Here is what we get currently:

![landmarks issue](https://user-images.githubusercontent.com/6945691/54566951-8ca0a500-498f-11e9-89f0-af899ad0e0a7.gif)

- [x] **1. Manifest title missing**  (#2237)

This appears to be drift. 

`<section ... aria-label="window">` was specified to be `<section ... aria-label="Window: [Manifest label]"` - This was working previously (back when the app bar was still at the end of the DOM). 

![landmarks ok](https://user-images.githubusercontent.com/6945691/54566956-91fdef80-498f-11e9-87bb-7e698a7030fe.gif)

- [x] **2. Sidebar is creating an extra "complementary" landmark (`<aside>`)** (#2246)

The content (panel) and navigation (tabs) of the left sidebar are meant, conceptually, to be one "thing". User opens the sidebar and navigates within it. It's been implemented as 2 things/drawers/whatever. We are close to visually masking this distinction, but to the screenreader, both components are identified as separate `<aside>` landmarks. I don't know whether the best approach is somehow wrapping them into one `<aside>`, or re-identifying the sidebar nav as `<nav>`, or something else altogether. @sdellis do you have an opinion here?

**Comment by camillevilla** [comment]
Explicitly assigning @sdellis here so that it pops up on his radar 🤞

**Comment by sdellis** [comment]
@jvine yes, I have an opinion here. The tabs are navigation elements and their content are asides, so I think it should look roughly like this: 
```
<div class="mirador-primary-window">
  <nav role="tablist" aria-labelledby="window-navigation">
    <h2 id="window-navigation">Window Navigation</h2>
    <button role="tab" aria-selected="true" aria-label="Open information companion window"><!-- icon goes here --></button>
	<button role="tab" aria-selected="false" aria-label="Open table of contents companion window"><!-- icon goes here --></button>
	...
  </nav> 
  <aside class="mirador-companion-area-left">Content for selected tab goes here... </aside>
</div>
